www/front_src/src/Resources/Details/tabs/Graph/index.tsx

```diff
@@ -78,6 +79,7 @@ const GraphTabContent = ({
         getIntervalDates={getIntervalDates}
         selectedTimePeriod={selectedTimePeriod}
         customTimePeriod={customTimePeriod}
+        navigateInGraph={navigateInGraph}
```

Comment:
we should rename this (and associated models) into "shiftTimePeriod"

Reply:
This is also used to zoom in graph, is this still relevant to rename it to "shiftTimePeriod" ?

Reply:
> This is also used to zoom in graph, is this still relevant to rename it to "shiftTimePeriod" ?

ahhh you’re right. What about "adjustTimePeriod" ?

Reply:
Purrfect 👌